Easydict/objc/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -382,6 +400,17 @@ - (EZQueryService *)defaultTTSService {
     return _defaultTTSService;
 }
 
+- (BOOL)isShowTipsView {
+    if (Configuration.shared.disableTipsView) {
+        return NO;
+    }
+    if (EZ_isEmptyString(self.queryModel.queryText) &&
+        !self.hasShowTips &&
+        self.queryModel.actionType != EZActionTypeInputQuery) {
```

Comment:
I think it's inappropriate to determine the timing of displaying the tips view here.

Firstly, I noticed that `isShowTipsView` is called multiple times. In theory, determining whether to display the tips view should only need to be done once to minimize misjudgments.

Secondly, this method relies on `queryText` and `actionType`, and during multiple calls to `isShowTipsView`, these two variables may change, especially since the result is refreshed and cleared with each query.

My suggestion is to determine whether to display the tips view at some appropriate point outside of `EZBaseQueryViewController`, then record it using a variable and pass the value in, rather than making the determination internally within `EZBaseQueryViewController`.